dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-http/src/main/java/org/apache/dolphinscheduler/plugin/alert/http/HttpSender.java

```diff
@@ -40,6 +38,11 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
```

Comment:
I am not sure whether we should do that or not, I think @SbloodyS @kezhenxu94 have discussed about this, we should change the code style unless we change the specific file

Reply:
This pr is reserved, and I will mention a new pr branch that only performs checkstyle processing on the files modified this time.

Reply:
> I am not sure whether we should do that or not, I think @SbloodyS @kezhenxu94 have discussed about this, we should change the code style unless we change the specific file

Yes. Modifying irrelevant files may lead to many conflicts in other pr. It's better only modify related files.